conftest.py

```diff
@@ -23,6 +23,12 @@ def staffuser():
     return UserFactory(short_name="Hello", password='password', is_staff=True)
 
 
+@pytest.fixture()
+def systemuser():
+    # Create it the same way the migration does it
+    return UserFactory(serial='__system__', full_name='System', password='!!')
```

Comment:
humm, at the end, why not making `add_user` (from the migration) an util we call from the tests too?